pkg/network/tracer/testutil/prefetch_file/prefetch_file.go

```diff
@@ -0,0 +1,32 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+)
+
+func main() {
+	if len(os.Args) != 3 {
+		fmt.Println("usage: prefetch_file <filename> <wait_time_in_seconds>")
+		return
```

Comment:
Would be better to return an non zero error code here (and other locations below) with `os.Exit`